tests/node-tests/basic-node-tests.js

```diff
@@ -39,8 +39,23 @@ fluid.require("test-module", require, "test-module");
 
 fluid.setLogging(true);
 
+// Set boolean for the  presence of the --tap command line option to output a
+// TAP report for consumption by testem when used as a custom launcher
+var shouldOutputTAP = process.argv.findIndex(function (argument) {
+    return argument.indexOf("--tap") > -1;
+}) > -1;
+
+// For accumulating TAP messages for parsing by testem custom launcher
+var tapOutput = "";
+
 QUnit.testDone(function (data) {
-    fluid.log("Test concluded - " + data.name + ": " + data.passed + " passed, " + data.failed + " failed");
+    var testMessage = "Test concluded - " + data.name + ": " + data.passed + " passed, " + data.failed + " failed";
+    fluid.log(testMessage);
+    if (shouldOutputTAP) {
```

Comment:
@the-t-in-rtf, I looked at this briefly after talking to @colinbdclark & we observed that `node-jqUnit` is actually not used in running these tests currently. So I think we could consider moving this feature upstream there, but it wouldn't address the immediate issue here of wanting to run the node packaging tests as part of the Infusion CI process.